front/src/app/navigation/map-view/map-view.component.ts

```diff
@@ -1,25 +1,25 @@
-import {AfterViewInit, Component, EventEmitter, Output, ViewChild} from '@angular/core';
+import {AfterViewInit, Component, EventEmitter, Output} from '@angular/core';
 import {DataService} from '../../services/data.service';
 import {CanvasResolutionConfigurator, TranslationPosition} from '../../services/CanvasResolutionConfigurator';
 import {ILocation, IMapItem, IPathNode} from '../../building-objects-if';
 import {IconOnMapRenderer} from '../../services/IconOnMapRenderer';
 import * as pip from 'point-in-polygon';
 import {CanvasTouchHelper} from '../../services/CanvasTouchHelper';
-import {IReceivedRoute} from "../../route-objects-if";
-import {MapItemRendererCanvas} from "./map-item-renderer.canvas";
-import {LocationRendererCanvas} from "./location-renderer.canvas";
-import {RouteRendererCanvas} from "./route-renderer.canvas";
-import {RendererProvider} from "./renderer-provider";
+import {IReceivedRoute} from '../../route-objects-if';
+import {MapItemRendererCanvas} from './map-item-renderer.canvas';
+import {LocationRendererCanvas} from './location-renderer.canvas';
+import {RouteRendererCanvas} from './route-renderer.canvas';
+import {RendererProvider} from './renderer-provider';
 
 @Component({
   selector: 'app-map-view',
   templateUrl: './map-view.component.html',
   styleUrls: ['./map-view.component.scss'],
 })
 export class MapViewComponent implements AfterViewInit {
-  private currentBuilding: string;
+  public currentBuilding: string;
```

Comment:
Evtl macht es hier Sinn in einem nächsten Schritt das von @sharpSteff vorgeschlagene Zustandsobjekt einzubauen, natürlich nur falls das passt :+1:

Reply:
ja das is da genau drin 👍

Reply:
Da müssen wir schauen, weil das ist ja der Zustand der MapView